build_tools/github/check_wheels.py

```diff
@@ -11,12 +11,16 @@
 build_matrix = wheel_config["jobs"]["build_wheels"]["strategy"]["matrix"]
 n_python_versions = len(build_matrix["python"])
 
-# For each python version we have: 7 wheels
+# Python 3.7, 3.8, and 3.9 each produces 7 wheels
 # 1 osx wheel (x86_64)
 # 4 linux wheel (i686 + x86_64) * (manylinux1 + manylinux2010)
 # 2 windows wheel (win32 + wind_amd64)
 n_wheels = 7 * n_python_versions
```

Comment:
should this be 
```suggestion
n_wheels = 7 * (n_python_versions - 1)
```
then?

Reply:
3.10 is not in the build matrix, so `n_python_veresions` is still 3.

I am starting to think that the build matrix is a bit too complex now, and it would be better to just list out all the configurations.

Reply:
I agree. An explicit list is probably better.

Reply:
Opened #21849 to explicitly list out the wheels.